src/dotdom.js

```diff
@@ -37,7 +37,7 @@ module.exports = window;
    * @param {Array} [children] - The child VNode elements
    * @returns {VNode} Returns a virtual DOM instance
    */
-  let createElement = (element, props={}, ...children) => (
+  let c = (element, props={}, ...children) => (
```

Comment:
I suppose the danger would only come if you used the build directly.. then it would be using the top level scope depending on how the file was added. In that case, I agree with you. I'll see if I can work in the scoping.